### ISSUE TITLE
Error installing SDWebImage

### ISSUE THREAD

**Issue by MrErkinjon** [SDWebImage/SDWebImage#3682]
### New Issue Checklist

* [ ] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [ ] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [ ] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios 
 Platform Version        | e.g. 12.0 / 10.14.0 / 12.0 / 5.0
 SDWebImage Version      | e.g. 5.19.0
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 15.2
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

[!] Error installing SDWebImage
[!] /usr/local/bin/git clone https://github.com/SDWebImage/SDWebImage.git /var/folders/d1/rn1jp5517hl96c_s0mgnhl4w0000gn/T/d20240225-11467-txsl7m --template= --single-branch --depth 1 --branch 5.19.0

Cloning into '/var/folders/d1/rn1jp5517hl96c_s0mgnhl4w0000gn/T/d20240225-11467-txsl7m'...
error: RPC failed; curl 92 HTTP/2 stream 5 was not closed cleanly: CANCEL (err 8)
error: 5654 bytes of body are still expected
fetch-pack: unexpected disconnect while reading sideband packet
fatal: early EOF
fatal: fetch-pack: invalid index-pack output

**Comment by dreampiggy** [comment]
Your network issue because of git clone failed
